server/reader_rocksdb.py

```diff
@@ -0,0 +1,120 @@
+from keynodes import KeynodeSysIdentifiers, Keynodes
+from sctp.logic import SctpClientInstance
+from sctp.types import ScAddr, SctpIteratorType, ScElementType
+from handlers import api_logic as logic
+
+import rocksdb
+import os
+import time
+import concurrent.futures
+
+from progress.bar import IncrementalBar
+
+class RocksdbReader:
+
+    addr_postfix = "_addrs"
+    long_content = None
+
+    def __init__(self):
+        self.sys = []
+        self.main = []
+        self.common = []
+        self.addr = []
+        self.long_content_counter = 0
+        self.content_len_max = 200
```

Comment:
Could you please describe 200 count?

Reply:
> Could you please describe 200 count?
Links with 100+ symbols have a high chance to drop exception while decoding it. 200 is an optimal value between performance and number of items to search, loss of items less than 1 percent.